core-bundle/src/Resources/contao/controllers/BackendConfirm.php

```diff
@@ -73,6 +73,12 @@ public function run()
 			$vars[$key] = $value;
 		}
 
+		// A valid back end request must point to a back end module
```

Comment:
I don‘t think this is correct. It could also be a custom URL using `key=` instead of `do=`, couldn‘t it?

Reply:
I don't think we have the request token check on `key=` calls

Reply:
also, `key=` replaces `act=`, you still need `do=`